backend/utils/llm.py

```diff
@@ -6,12 +7,12 @@
 logger = logging.getLogger(__name__)
 config = Config()
 
-client = MistralClient(api_key=config.mlarge_key, endpoint=config.mlarge_url)
```

Comment:
Forgive me if I'm missing something, but why do we no longer need the endpoint argument?

Reply:
As we are using the normal mistral api for this now opposed to the azure deployment. This is due to the function calling features that are not available in the azure space at this time. It therefore doesn't require the endpoint